### PR TITLE
build: work around sccache false positives due to asm .incbin usage

### PR DESCRIPTION
**Pull request by piscisaureus**

### CODE REVIEW

File: BUILD.gn [hunk 1]
```diff
@@ -241,6 +263,29 @@ run_node("bundle") {
...
+  inputs = get_target_outputs(":bundle")
+  outputs = [
+    "$target_gen_dir/bundle/hash.h",
+  ]
+  deps = [
+    ":bundle",
+  ]
+  args = [
+    "--format",
+    "__attribute__((__unused__)) static const int dummy_%s = 0;",
```
Comment (ry) [inline BUILD.gn]: If you just use a comment, does it not work?
```
"/* %s */"
```

File: BUILD.gn [hunk 2]
```diff
@@ -102,6 +103,27 @@ static_library("libdeno") {
...
+  # header file that contains the the sha256 hash of the snapshot.
+  if (cc_wrapper != "" && cc_wrapper != "ccache") {
+    hash_h = "$target_gen_dir/bundle/hash.h"
+    inputs += [ hash_h ]
+    deps += [ ":bundle_hash_h" ]
+    if (is_win) {
+      cflags = [ "/FI" + rebase_path(hash_h, target_out_dir) ]
+    } else {
+      cflags = [
+        "-include",
```
Comment (ry) [inline BUILD.gn]: side comment - we should really support single dash flags in our own flags.

Comment (piscisaureus) [reply BUILD.gn]: No, comments are removed by the preprocessor.

File: BUILD.gn (hunk 1 above)
Comment (piscisaureus) [reply BUILD.gn]: No, comments are removed by the preprocessor.

**Pull request merged**